packages/swapper/src/swappers/ThorchainSwapper/utils/getLongtailRate.ts

```diff
@@ -79,11 +81,11 @@ export const getLongtailToL1Rate = async (
     sellAmountIncludingProtocolFeesCryptoBaseUnit: quotedAmountOut.toString(),
   }
 
-  const thorchainRates = await getL1Rate(
+  const thorchainRates = await getL1RateOrQuote<ThorTradeRate>(
     l1Tol1QuoteInput,
     deps,
     streamingInterval,
-    TradeType.LongTailToL1,
```

Comment:
And here too, same issue with hardcoded `TradeType.L1ToL1` tradeType

Reply:
13e5465b7a1273b360e2a3781c87ae51ff1e3d57